lms/static/js/student_account/views/account_settings_factory.js

```diff
@@ -0,0 +1,215 @@
+;(function (define, undefined) {
+    'use strict';
+    define([
```

Comment:
@bryanlandia this is definitely comprehensive theming.  I've got a whole series of scars to prove it.  I'll give these changes a shot ;)